src/main/java/com/arpnetworking/clusteraggregator/aggregation/StreamingAggregator.java

```diff
@@ -306,6 +309,7 @@ private String createHost() {
     private String _metric;
     private String _service;
     private AggregatedData.Builder _resultBuilder;
+    private DateTime _latestClosedBucket = DateTime.now().minusHours(1);
```

Comment:
This seems arbitrary. If we are recovering from an outage and MAD has buffered data, this could artificially limit the back-fill. See my general comment.